src/components/InputField/InputField.tsx

```diff
@@ -30,6 +37,8 @@ export default function InputField({
       secureTextEntry={secureTextEntry}
       autoCorrect={false}
       autoCapitalize="none"
+      keyboardType={inputMode}
```

Comment:
`inputMode` is also a prop on `TextInput` right? you could probably use that directly